.evergreen/docker/ubuntu20.04/base-entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -eu
+
+rm -f $DRIVERS_TOOLS/results.json
+cd $DRIVERS_TOOLS/.evergreen
+bash run-orchestration.sh
+
+# Change permissions of files we have created.
```

Comment:
Suggest tweaking this comment to state we are "restoring" (or "preserving") the permissions of Docker-owned files to those of the user (document that this is to address potential Docker vs. user permissions discrepancies).

Reply:
Done.